src/Message/Launch/Validator/AbstractLaunchValidator.php

```diff
@@ -51,6 +51,12 @@ abstract class AbstractLaunchValidator implements LaunchValidatorInterface
     /** @var string[] */
     protected $successes = [];
 
+    /** @var bool */
+    protected bool $isStateValidationRequired = true;
```

Comment:
Why would you not validate state?

Reply:
https://github.com/oat-sa/lib-lti1p3-core/pull/161#discussion_r1196325086

Reply:
why did you add it to abstract validator yet it seems to be a workaround that is needed only for a tool validator?

Reply:
I believe it would be convenient in case we need to give access to the portal on another resource. I also checked open connect specification and both parameters (state and nonce) are not mandatory, they recommended and optional.